ariac-competitor/build_competitor_image.bash

```diff
@@ -2,8 +2,22 @@
 
 DOCKER_ARGS=""
 # Uncoment this line to rebuild without cache
+# TODO: expose this as an argument
```

Comment:
I'm using the `no-cache` option in my upcoming `kinetic_version` branch, let's handle it from there.